doc/ExtraDocuments/Fileformats.md

```diff
@@ -0,0 +1,248 @@
+# File formats of Magnetic field data
+
+In this section we explain the different file formats that one might encounter when dealing with magnetic field data. We state in which context they usually appear, as well as codes generating it/converting it to other formats. 
+
+For syntax: The name of each section is given by the file format (i.e. how the data is structured/arranged in the file). In paranthese one finds the type of field (2D or 3D) that is usually stored, as well as a common code that generates the files in question. Finally the file type (different from format) i.e. the `.extension` of the file (e.g. textfile, binary netcdf, ...) is listed as well.
+
+## gfiles/EQDSK format (axissymmetric 2D equillibrium, EFIT, textfile)
+
+[Original documentation from 1997](https://w3.pppl.gov/ntcc/TORAY/G_EQDSK.pdf) (note that there are some mistakes, e.g. `rcentr` does not exist) <br>
+
```

Comment:
What do you mean, `rcentr` doesn't exist? `field_divB0` uses it when `fpol` is not used.

Reply:
Nevermind, that was a mistake on my part. I meant this supposedly variable `xdum`, but at this point I deem it is just a place filler not an actual variable.